packages/server/src/__tests__/plugin/drainHttpServer/stoppable.test.ts

```diff
@@ -135,8 +134,10 @@ Object.keys(schemes).forEach((schemeName) => {
             scheme.agent({ keepAlive: true }),
           ),
         );
-        expect(err.message).toMatch(/ECONNREFUSED/);
-        expect(closed).toBe(0);
+        expect(err.code).toMatch(/ECONNREFUSED/);
+
+        const isNode20 = !!process.version.match(/^v20\./);
+        expect(closed).toBe(isNode20 ? 1 : 0);
```

Comment:
(currently) unexplained change in behavior in `http(s).Server`, but maybe this isn't actually important for us to validate/enforce. https://github.com/nodejs/node/pull/46333 seems to be the only relevant change listed in the Node 20 changelog but I haven't been able to apply the information there in a useful way that "corrects" this test. That PR also only seems to apply to when expected headers don't exist, which isn't the case here.

Reply:
Or https://github.com/nodejs/node/pull/46331 ?

Reply:
For future reference, @glasser helped me pin this to changes made in v19 and v20.3
(http) https://github.com/nodejs/node/pull/43522
(https equiv) https://github.com/nodejs/node/pull/48383